AppAuth.podspec

```diff
@@ -33,16 +33,21 @@ It follows the OAuth 2.0 for Native Apps best current practice
   s.platforms    = { :ios => "7.0", :osx => "10.9", :watchos => "2.0", :tvos => "9.0" }
 
   s.source       = { :git => "https://github.com/openid/AppAuth-iOS.git", :tag => s.version }
-
-  s.source_files = "Source/*.{h,m}"
```

Comment:
Suggest reverting this (see below).

Reply:
due to the cocoapods error for empty subspecs (refer to my other comment) I had to add at least something to the subspec, so I decided to move the `source_files` into the `Core` subspec...